src/other/PropertyEditor.js

```diff
@@ -247,11 +251,42 @@
         }
     };
 
+    PropertyEditor.prototype.filterInputs = function(d) {
+        var discArr = Persist.discover(d);
+        if ((this.filterTags() || this.excludeTags().length > 0 || this.excludeParams.length > 0) && d instanceof PropertyEditor === false) {
+
+            var context = this;
+            return discArr.filter(function(param, idx) {
+                if (context.excludeParams().length > 0) {
+                    for (var i = 0; i < context.excludeParams().length; i++) {
+                        var widgetName = context.excludeParams()[i].split(".")[0];
+                        var excludeParam = context.excludeParams()[i].split(".")[1];
+                        if (d.class().indexOf(widgetName) !== -1) {
+                            if (param.id === excludeParam) {
+                                return false;
+                            }
+                            return true;
+                        }
+                    }
+                }
+                if (context.excludeTags().length > 0 && param.ext && param.ext.tags && param.ext.tags.filter(function(item){ return (context.excludeTags().indexOf(item) > -1); }).length > 0) {
```

Comment:
@GordonSmith can you elaborate on this .some returns true/false ... disArr is an array of params